gradio/components.py

```diff
@@ -2521,15 +2528,23 @@ def postprocess(self, y):
             return y
         if isinstance(y, str):
             y = pd.read_csv(str)
```

Comment:
Unrelated to your PR but this is a bug right? It should be `y = pd.read_csv(y)` right?

Reply:
It has to be. I'll fix in this PR.